compiler/resolution/functionResolution.cpp

```diff
@@ -8687,7 +8687,7 @@ buildVirtualMaps() {
       continue;
 
     if (fn->hasFlag(FLAG_NO_PARENS))
-      // Parenthesesless functions are statically bound; that is, they are not
+      // Parentheses-less functions are statically bound; that is, they are not
```

Comment:
I submitted this separately as #4929

I'll shortly repush this branch with this change elided, among a few other changes.  (And rebased on top of current master rather than trying to finagle the parentheses-less commit out from behind the merge from master.)